examples/connect/lambdafunctionassociation.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: connect.aws.upbound.io/v1beta1
+kind: LambdaFunctionAssociation
+metadata:
+  annotations:
+    meta.upbound.io/example-id: connect/v1beta1/lambdafunctionassociation
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    functionArnSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    instanceIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    region: us-east-1
+---
+apiVersion: lambda.aws.upbound.io/v1beta1
+kind: Function
+metadata:
+  annotations:
+    meta.upbound.io/example-id: connect/v1beta1/lambdafunctionassociation
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    region: us-east-1
+    environment:
+    - variables:
+        foo: bar
+    s3Bucket: upbound-provider-test-data
+    s3Key: hello-python.zip
+    handler: index.py
+    roleRef:
+      name: sample-role
+    runtime: python3.9
+---
+apiVersion: iam.aws.upbound.io/v1beta1
+kind: Role
+metadata:
```

Comment:
could you please add the following annotation:
```
annotations:
    meta.upbound.io/example-id: connect/v1beta1/lambdafunctionassociation
```

Reply:
I have added the specified annotation.